00-flex.yaml

```diff
@@ -46,6 +46,10 @@ template: |
             set -x -e
 
             curl -s https://get.symfony.com/cloud/configurator | (>&2 bash)
+            {{ range $ext := php_extensions -}}
+            {{- if not php_extension_available $ext $.PhpVersion -}}
+            (>&2 php-ext-install {{ $ext }} version)
```

Comment:
I would remove "version" here so that the command works out of the box without any changes (same below).

Reply:
I won't as a version is required

Reply:
Let's comment the command then?

Reply:
you mean like this:
```suggestion
            # (>&2 php-ext-install {{ $ext }} version)
```
?

Reply:
or like this maybe:
```suggestion
            # (>&2 php-ext-install {{ $ext }} X.Y.Z)
```
?

Reply:
Second one